modules/webbrowser/src/properties/propertycefsynchronizer.cpp

```diff
@@ -114,7 +114,7 @@ bool PropertyCefSynchronizer::OnQuery(CefRefPtr<CefBrowser> browser, CefRefPtr<C
         if (nextValPos == std::string::npos) {
             LogWarn("Missing enclosing } in: " + requestStr);
         } else {
-            auto message = requestStr.substr(0, nextValPos);
+            auto message = requestStr.substr(0, nextValPos + 1);
```

Comment:
you should really use out json library to do the json parsing... look in ext/json...

Reply:
Agree, but that would be another PR